libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -969,7 +747,7 @@ void AC_AutoTune::load_orig_gains()
 {
     attitude_control->bf_feedforward(orig_bf_feedforward);
     if (roll_enabled()) {
-        if (!is_zero(orig_roll_rp)) {
+        if (!is_zero(orig_roll_rp) || (is_zero(orig_roll_rp) && allow_zero_rate_p())) {
```

Comment:
I think there's no need for the second "is_zero(orig_roll_rp)".  The check can be, "if (!is_zero(orig_roll_rp) || allow_zero_rate_p()) {"

Reply:
makes sense. thanks!